templates/Template.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { TextTemplate } from './TextTemplate'
+import { TextTutorialTemplate } from './TextTutorialTemplate'
+import { MenuTemplate } from './MenuTemplate'
+
+import { lessons } from '../courses/CourseAbc'
+
+const templateMap = new Map()
```

Comment:
Why do we need a map? Currently, this `Map` will be set once, anyway. So why don't you use a plain object:
```
export const Text = TextTemplate
export const TextTutorial = TextTutorialTemplate
export const Menu = MenuTemplate
```
Then somewhere else:
```
import * as templates from './templates/Template.js'

const id = 'Text'
const template = templates[id]
```